src/xenia/emulator.cc

```diff
@@ -114,9 +114,15 @@ Emulator::Emulator(const std::filesystem::path& command_line,
   // show the quickstart guide the first time they ever open the emulator
   uint64_t persistent_flags = GetPersistentEmulatorFlags();
   if (!(persistent_flags & EmulatorFlagQuickstartShown)) {
+#if XE_PLATFORM_WIN32 == 1
+  if (MessageBoxW(nullptr, L"Xenia does not support or condone piracy in anyway shape or form\nDo you want to open the quickstart quide?", L"Xenia", MB_YESNO | MB_ICONQUESTION) == IDYES) {
```

Comment:
There's two typos here: there should be a full stop after the end of the first sentence (before the newline) and it should be "guide", not "quide".